webapp/src/components/link_embed_preview/embed_preview.scss

```
@@ -0,0 +1,157 @@
+$light-gray: #6a737d;
+$light-blue: #eff7ff;
+$github-merged: #6f42c1;
+$github-closed: #cb2431;
+$github-open: #28a745;
+$github-not-planned: #6e7681;
+
+@media (min-width: 544px) {
+    .github-preview--large {
+        min-width: 320px;
+    }
+}
+
+/* Github Preview */
+.github-preview {
+    background-color: var(--center-channel-bg-rgb);
+    box-shadow: 0 2px 3px rgba(0,0,0,.08);
+    position: relative;
+    width: 100%;
+    max-width: 700px;
+    border-radius: 4px;
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+
+    /* Header */
+    .header {
+        color: rgba(var(--center-channel-color-rgb), 0.64);
+        font-size: 11px;
+        line-height: 16px;
+        white-space: nowrap;
+
+        a {
+            text-decoration: none;
+            color: rgba(var(--center-channel-color-rgb), 0.64);
+            display: inline-block;
+    
+            .repo {
+                color: var(--center-channel-color-rgb);
+            }
+        }
+    }
+    
+    /* Body */
+    .body > span {
+        line-height: 1.25;
+    }
+    
+    /* Info */
+    .preview-info {
+        line-height: 1.25;
+        display: flex;
+        flex-direction: column;
+
+        > a,
+        > a:hover {
+            display: block;
+            text-decoration: none;
+            color: var(--link-color);
+        }
+
+        > a span {
+            color: var($light-gray);
+        
+            h5 {
+                font-weight: 600;
+                font-size: 14px;
+                display: inline;
+            
+                span.github-preview-icon-opened {
+                    color: var($github-open);
```

These scss variables shouldn't use `var` right?

```suggestion
                    color: $github-open;
```